### PR TITLE
Niloofar/ Added tests for MobileMenu component

### PR DESCRIPTION
**Pull request by niloofar-deriv**
This PR includes:

- MobileMenu component tests

### CODE REVIEW

**Review by ameerul-deriv — APPROVED** [review]

File: src/components/AppHeader/MobileMenu/__tests__/MobileMenu.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+const MobileMenuComponent = () => (
+    <BrowserRouter>
+        <QueryParamProvider adapter={ReactRouter5Adapter}>
+            <MobileMenu />
+        </QueryParamProvider>
+    </BrowserRouter>
+);
+
+describe('MobileMenu component', () => {
+    beforeEach(() => {
```
Comment (nada-deriv) [inline src/components/AppHeader/MobileMenu/__tests__/MobileMenu.spec.tsx:41]: is this needed? since its already mocked above

**Review by ali-hosseini-deriv — APPROVED** [review]

File: src/components/AppHeader/MobileMenu/__tests__/MobileMenu.spec.tsx (hunk 1 above)
Comment (niloofar-deriv) [reply src/components/AppHeader/MobileMenu/__tests__/MobileMenu.spec.tsx:41]: Yep nada we need them
at the top I mocked the hooks here I'm mocking the returning value

Comment (niloofar-deriv) [reply src/components/AppHeader/MobileMenu/__tests__/MobileMenu.spec.tsx:41]: If you have a better suggestion for implementing this please let me know I'll change it

Comment (nada-deriv) [reply src/components/AppHeader/MobileMenu/__tests__/MobileMenu.spec.tsx:41]: so can we not directly mock the return values above itself? will there be a difference?

Comment (niloofar-deriv) [reply src/components/AppHeader/MobileMenu/__tests__/MobileMenu.spec.tsx]: @nada-deriv I will make some changes regarding this 🙏

**Review by nada-deriv — APPROVED** [review]

**Pull request merged**